frontends/web/src/routes/account/send/send.tsx

```diff
@@ -166,6 +166,18 @@ class Send extends Component<Props, State> {
           break;
         }
         break;
+      case 'backend':
+        switch (data) {
+        case 'newTxs':
```

Comment:
this is the wrong event, you want `syncdone` like in account.tsx.

Reply:
Changed, thanks!